o11y/tracer/types/types.go

```diff
@@ -0,0 +1,24 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+package types
+
+// SinkConfig describes a config for a tracer/profiler sink config
+type SinkConfig struct {
+	EnableTracer   bool
+	EnableProfiler bool
+	SampleRate     float64
+}
+
+// SinkDriver represents a sink driver to use
+type SinkDriver string
+
+const (
+	// SinkDriverDatadog is the Datadog driver
+	SinkDriverDatadog SinkDriver = "datadog"
```

Comment:
I think the reason is to ease the config flag setup; flag/configmap input is a string, and that allows to easily follow through to the option in the code. It's the pattern used for the metrics sink and consistency can be appreciated.
Sure it could be done another way, but this works decently in my opinion.

Reply:
Yes indeed, I did not make the correlation with the configuration file, hence string is appropriate

Thanks!